lib/internal/cluster/master.js

```diff
@@ -70,9 +70,9 @@ cluster.setupMaster = function(options) {
   assert(schedulingPolicy === SCHED_NONE || schedulingPolicy === SCHED_RR,
          `Bad cluster.schedulingPolicy: ${schedulingPolicy}`);
 
-  const hasDebugArg = process.execArgv.some((argv) => {
-    return /^(--debug|--debug-brk)(=\d+)?$/.test(argv);
-  });
+  const hasDebugArg = process.execArgv.some(
+    (argv) => /^(--debug|--debug-brk)(=\d+)?$/.test(argv)
```

Comment:
No, linter doesn't force it. I was just doing what I thought was easiest to grok, which was to put the entire arrow function on a single line. Happy to tweak it if it's the difference between "no way" and "I can live with this PR".